### ISSUE TITLE
Problem converting unsigned hex literals in C

### ISSUE THREAD

**Issue by EmileBV** [jesperjohansson/vscode-hex-multi-converter#1]
In C/C++, it's very common to add `U` (or `u`) at the end of a number to tell the compiler the number should be interpreted as unsigned. This extension doesn't work for those cases.
Exemple:
```c
// 1- Select the following literals:
0x12345U
0x12345

// 2- Call "Convert Hex to Decimal" command
// Result:
0x12345U
1192517
```

**Comment by jesperjohansson** [comment]
@EmileBV I have published a new version v0.4.0 which adds a new setting for enabling "extended matcher".

To test it out, you can add this to your settings.json after updating the extension:
```
"hex-multi-converter.extendedMatcher": true
```

**Comment by EmileBV** [comment]
That was fast! I just updated the plugin, works great, thanks a lot!
